src/client/Microsoft.Identity.Client/Platforms/netcore/NetCorePlatformProxy.cs

```diff
@@ -201,10 +201,12 @@ public override IPoPCryptoProvider GetDefaultPoPCryptoProvider()
         {
             return PoPProviderFactory.GetOrCreateProvider();
         }
-        public override IBroker CreateBroker(CoreUIParent uiParent)
```

Comment:
There is a base class that provides an implementation for this, which simply throws a not supported exceptions. Customers do not see this exception, because they cannot enable the broker on .NET, as we do not expose `.WithBroker`